code/modules/mob/language.dm

```diff
@@ -757,6 +757,8 @@
 	desc = "Bark bark bark."
 	key = "vu"
 
-
+/mob/proc/grant_all_languages()
+	for(var/la in GLOB.all_languages)
+		src.add_language(la)
```

Comment:
This doesn't grant restricted languages, right? Such as the Shadowling hivemind?

Reply:
Probably does because all_ is legit everything, maybe we can have another list under glob called all_unrestricted?

Reply:
I was told that it was meant to grant everything, if that is not the case it can be changed though.

Reply:
That's what I thought too 🤔

Reply:
All good then.